internal/v1/oscap.go

```diff
@@ -73,37 +78,29 @@ func OscapProfiles(distribution Distributions) (DistributionProfileResponse, err
 	}
 }
 
-func OscapDistributionGeneralization(distribution Distributions) (string, error) {
-	switch distribution {
-	case Rhel8:
-		fallthrough
-	case Rhel84:
-		fallthrough
-	case Rhel85:
-		fallthrough
-	case Rhel86:
-		fallthrough
-	case Rhel87:
-		fallthrough
-	case Rhel88:
-		fallthrough
-	case Rhel8Nightly:
-		return "rhel8", nil
-	case Rhel9:
-		fallthrough
-	case Rhel90:
-		fallthrough
-	case Rhel91:
-		fallthrough
-	case Rhel92:
-		fallthrough
-	case Rhel9Nightly:
-		return "rhel9", nil
-	case Centos8:
-		return "centos8", nil
-	case Centos9:
-		return "cs9", nil
-	default:
-		return "", errors.New("No customizations for the specified distribution")
+func loadOscapCustomizations(distributionDir string, distribution Distributions, profile DistributionProfileItem) (*Customizations, error) {
```

Comment:
My suggestion is to load the oscap customizations when we load [the distributions](https://github.com/osbuild/image-builder/blob/3e09617ce00040a5d4c61fc4948c963719f5c6e2/internal/distribution/distribution.go#L165) on startup and just save them as a map in [`DistributionItem`](https://github.com/osbuild/image-builder/blob/3e09617ce00040a5d4c61fc4948c963719f5c6e2/internal/distribution/distribution.go#L18).

This has several advantages:
- No I/O in the handler.
- No passing `distributionDir` to the server.
- The customization's existence and validity (if it's a valid JSON) is checked during launch. This means that if there's something up with them, the container will crash immediately instead of behaving weirdly during runtime.

The disadvantage is higher memory usage, but they are tiny, this is fine.